README.md

```diff
@@ -90,6 +90,39 @@ Runtime Environment (node v10.15.1 on linux)
 CLI version 3.0.1
 ```
 
+## Install locally via npm or yarn (alternative install)
+
+Required: current version of yarn or npm installed
+
+Create a file `package.json` within your project with the following content:
+
+```javascript
+{
+  "name": "example",
+  "version": "1.0.0",
+  "dependencies": {
+    "asciidoctor": "^2.0.3",
+    "asciidoctor-pdf": "^1.0.0-alpha.3"
+  },
+  "scripts": {
+    "render-pdf": "asciidoctor-pdf <path to your adoc-file>"
+  }
+}
+```
+
+The commands
+
+```
+yarn install
+yarn render-pdf
+```
+
+will then install `asciidoctor` and `asciidoctor-pdf` as local module and render your AsciiDoc file as PDF to the same folder as your soruce file is located.
+
+To use npm instead of yarn, use `npm install` and `npm run render-pdf` instead.
+
+Note: this is verified to work on Windows in WSL.
```

Comment:
Admonition does not exist in Markdown but could you please use the following syntax for consistency:

```suggestion
**NOTE:** This is verified to work on Windows in WSL.
```

Reply:
btw: why don't you convert the `README.md` to `README.adoc`?

Reply:
I wish... but npmjs.org only supports Markdown :disappointed: